File: projects/challenges/programmingChallenge/python/day6Calculator.py

def division(x, y):
    print(x / y)

def multiplication(x, y):
    print(x * y)

def substract(x, y):
    print(x - y)

def add(x, y):
    print(x + y)
    

def calculator(num1, operator, num2):
    
    if operator == "+":
        add(num1, num2)

    elif operator == "-":
        substract(num1, num2)

    elif operator == "*":
        multiplication(num1, num2)

    elif operator == "/":
        division(num1, num2)

    else:
        print("Your operator isn't correct. Try again.")
        


if __name__ == "__main__":
    print("""
    Hi! Welcome to the calculator.
    Here, you'll be able to add, substract, multiply or divide two numbers.
    
    ***OPERATORS***
    Use + to add.
    Use - to substract.
    Use * to multiply.
    Use / to divide.
    
    """)
    num1 = int(input("Write the first number: "))
    operator = str(input("Write the operator: "))
    num2 = int(input("Write the second number: "))
    calculator(num1, operator, num2)

File: projects/challenges/programmingChallenge/js/day6Calculator.js
/* The challenge consists on creating a program
able to work as if it were a calculator. */

function Calculator(num1, operator, num2) {
  switch (operator) {
    case "+":
      return num1 + num2;
      break;

    case "-":
      return num1 - num2;
      break;

    case "*" /* || "x" || "X" */:
      return num1 * num2;
      break;

    case "/":
      return num1 / num2;
      break;

    case "x":
      return "If you want to multiply, you may use '*'.";
      break;

    case "X":
      return "If you want to multiply, you may use '*'.";
      break;

    default:
      return "Your operator is incorrect. Try again.";
      break;
  }
}

console.log(Calculator(2, "+", 2));
console.log(Calculator(4, "/", 2));
console.log(Calculator(8, "*", 4));
